docs/auto-instrumentation-install.md

```diff
@@ -403,6 +410,40 @@ provides best effort support with issues related to native OpenTelemetry instrum
 | apache-httpd            | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-apache-httpd-instrumentation) | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
 | nginx                   | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-apache-httpd-instrumentation) | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
 
+### CRD Management
+
+When deploying the operator, the required Custom Resource Definitions (CRDs) must be deployed beforehand.
+
+#### Recommended Approach: Automated CRD Deployment
+
+Set the Helm chart value `operatorcrds.install=true` to allow the chart to handle CRD deployment automatically.
+_This option deploys the CRDs using a local subchart, available at https://github.com/signalfx/splunk-otel-collector-chart/tree/main/helm-charts/splunk-otel-collector/charts/opentelemetry-operator-crds._
+
+#### Alternative Approach: Manual CRD Deployment
+
+If you prefer to manage CRD deployment manually, apply the CRDs using the commands below before installing the Helm chart:
```

Comment:
How likely is it that these commands get outdated? Is there any way to make this into a test to ensure this doesn't go stale?

_This is not a blocker for this PR, just something to consider._

Reply:
Maybe at most one or twice a year. I think we need to make supporting CRD versioning and upgrades overtime a separate ticket though.